README.md

```diff
@@ -1,15 +1,28 @@
-# Calendar-Proxy
+# TUM Calendar Proxy
 
+This is a proxy service that simplifies and enhances the iCal export from TUM Online. It allows you to:
 
-Proxy for the TUM iCal export to remove clutter from it and optimize the output
+- Shorten long lesson names, such as 'Grundlagen Betriebssysteme und Systemsoftware' → 'GBS'
+- Add locations that are recognized by Google Maps / Google Now
```

Comment:
I don't think google now is relevant any more.
What do you think about this?

```suggestion
- Add locations that are recognized by Google / Apple Maps
```

Reply:
Yeah for sure